web_interface/astpp/application/libraries/csvreader.php

```diff
@@ -103,7 +103,8 @@ function parse_lines($p_CSVLines, $config_variable, $check_header_flag = false)
 					$value = strip_slashes ( trim ( $value ) );
 					$value = preg_replace ( '#<script.*</script>#is', '', $value );
 					if (isset ( $field_key_value ) && ! empty ( $field_key_value ))
-						$content [$field_key_value] = strip_tags ( filter_var ( $value, FILTER_SANITIZE_STRING ) );
+//						$content [$field_key_value] = strip_tags ( filter_var ( $value, FILTER_SANITIZE_STRING ));
```

Comment:
We need this line here as some time we are getting html text in value. So we want to clean them before processing.

Reply:
Some translations has html tags (for examle "Connection <br>Cost"). Installing laguage addon pass througth this module and filtering some strings like this. I think this bad practice, used html tags in translation, better use css-style for word wrap in tables header. But now for minimum code correction remove this filter from csvreader library.